content/en/docs/08/labs/81.md

```diff
@@ -32,8 +33,7 @@ Apply it using the following command:
 kubectl -n application-metrics apply -f ~/work/servicemonitor.yaml
 ```
 
-Verify that the target gets scraped in the [Prometheus user interface](http://LOCALHOST:19090/targets). Target name: `application-metrics/example-web-python-monitor/0` (It may take up to a minute for Prometheus to load the new
-configuration and scrape the metrics).
+Verify that the target gets scraped in the [Prometheus user interface](http://LOCALHOST:19090/targets). Target name: `serviceMonitor/application-metrics/example-web-python-monitor/0` (It may take up to a minute for Prometheus to load the new configuration and scrape the metrics).
```

Comment:
```suggestion
Verify that the target gets scraped in the [Prometheus user interface](http://LOCALHOST:19090/targets). Target name: `serviceMonitor/application-metrics/example-web-python-monitor/0` (it may take up to a minute for Prometheus to load the new configuration and scrape the metrics).
```